hphp/system/php/spl/datastructures/SplObjectStorage.php

```diff
@@ -364,7 +364,8 @@ public function unserialize($serialized) {
    * @return     mixed   No value is returned.
    */
   public function setInfo($data) {
-    current(&$this->__storage)['inf'] = $data;;
+    $storage = current(&$this->__storage);
```

Comment:
Was this already broken, or is there a way to make this syntax work in PHP Code?

Reply:
I reenabled the syntax in the second commit but forgot to revert this change. Will revert this change.